### ISSUE TITLE
Svelte transitions should be disabled when motion is reduced

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1236]
Not sure how I forgot about this, but when you click the CW button, there is an animation where it slides up. That should be removed when we're reducing motion.
